api/opds.py

```diff
@@ -721,6 +721,10 @@ def language_and_audience_key_from_work(self, work):
             audiences = Classifier.AUDIENCES_JUVENILE
         elif work.audience in Classifier.AUDIENCES_ADULT:
             audiences = list(Classifier.AUDIENCES)
```

Comment:
Should this  be list(AUDIENCES_NO_RESEARCH)? I don't think we want research titles showing up in a "by this author" list that's not research-related. But maybe we do!

Reply:
We can always change it later.

Reply:
`AUDIENCES_NO_RESEARCH` makes more sense then.